src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -276,7 +274,9 @@ def load_performance_issues(self, light: bool, snuba_params: ParamsType | None)
                 if self.nodestore_event is not None:
                     occurrence_query = DiscoverQueryBuilder(
                         Dataset.IssuePlatform,
-                        snuba_params,
+                        # Params is ignored if snuba_params is passed
+                        params={},
```

Comment:
I assume this is because `params` is a positional argument in the query builder still? It'd be nice if it was an optional one in the future to avoid this strange signature.

Reply:
Changing that breaks a buuuunch of stuff lol, will be doing that in the future but increases the impact of the change significantly